src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,32 @@
+.nav {
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    position: relative;
+    padding: 23px 0;
+
+    @include textDecorationUpperStandart;
+    color: $main-text-color;
+    font-size: 12px;
+    line-height: 14px;
+
+    &--hover-blue:hover {
```

Comment:
use `hover` for `&__link`

Reply:
i use it for &__link (but with modaficator, if i want to change the color), i think im ok with it